docs/source/bricks.rst

```diff
@@ -753,29 +753,58 @@ You can disable ``max_partition`` by setting it to ``None``.
 ``partition_via_api`` allows users to partition documents using the hosted Unstructured API.
 The API partitions documents using the automatic ``partition`` function. Currently, the API
 supports all filetypes except for RTF and EPUBs.
```

Comment:
We still don't support these in the API?

Reply:
I think we do now, @yuming-long bumped the `pandocs` version a while back. Will update that.